admin/controllers/school.js

```diff
@@ -44,15 +44,23 @@ const getPupils = async (req, res, next) => {
   const order = JSON.parse(sortOrder)
   res.locals.sortOrder = typeof order === 'boolean' ? !order : true
   res.locals.sortClass = order === false ? 'sort up' : 'sort'
+
   let pupilsFormatted
+  let groupsIndex = []
+
+  try {
```

Comment:
wouldn't it make more sense to place the single line this await call(below) within the try catch block below?
that would also reduce the code a bit.